geodiff/tests/GTestCMakeLists.in

```diff
@@ -8,7 +8,7 @@ PROJECT(googletest-download NONE)
 INCLUDE(ExternalProject)
 EXTERNALPROJECT_ADD(googletest
     GIT_REPOSITORY https://github.com/google/googletest.git
-    GIT_TAG release-1.8.1
+    GIT_TAG release-1.12.1
```

Comment:
just curious, was this needed for something?

Reply:
Not needed, leftover from some experiments.

Reply:
Hmm apparently the original version of gtest emits a compilation warning (treated as an error) with compiler on ubuntu 22.04. When I switch back to 1.12.1 you had there, there's no error...
```
In file included from /home/martin/lutra/geodiff/build-geodiff-Desktop-Debug/googletest-src/googletest/src/gtest-all.cc:42:
/home/martin/lutra/geodiff/build-geodiff-Desktop-Debug/googletest-src/googletest/src/gtest-death-test.cc: In function ‘bool testing::internal::StackGrowsDown()’:
/home/martin/lutra/geodiff/build-geodiff-Desktop-Debug/googletest-src/googletest/src/gtest-death-test.cc:1224:24: error: ‘dummy’ may be used uninitialized [-Werror=maybe-uninitialized]
 1224 |   StackLowerThanAddress(&dummy, &result);
      |   ~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~
/home/martin/lutra/geodiff/build-geodiff-Desktop-Debug/googletest-src/googletest/src/gtest-death-test.cc:1214:13: note: by argument 1 of type ‘const void*’ to ‘void testing::internal::StackLowerThanAddress(const void*, bool*)’ declared here
 1214 | static void StackLowerThanAddress(const void* ptr, bool* result) {
      |             ^~~~~~~~~~~~~~~~~~~~~
/home/martin/lutra/geodiff/build-geodiff-Desktop-Debug/googletest-src/googletest/src/gtest-death-test.cc:1222:7: note: ‘dummy’ declared here
 1222 |   int dummy;
      |       ^~~~~
cc1plus: all warnings being treated as errors
```

Reply:
I'm still on 20.04, so haven't seen this. Bumped gtest version to 1.12.1.